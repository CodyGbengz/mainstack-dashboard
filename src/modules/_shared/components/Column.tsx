import styled from 'styled-components';

export const Column = styled.div`
  flex-flow: column wrap;
  display: flex;
  gap: 1rem;
`;