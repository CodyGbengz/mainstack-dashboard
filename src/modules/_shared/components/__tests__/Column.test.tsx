import { render } from '@testing-library/react';
import { Column } from '../Column';

describe('COMPONENT TEST: Column', () => {
  const setup = () => render(<Column />);

  test('Should match snapshot', () => {
    const { container } = setup();
    expect(container.parentElement).toMatchSnapshot();
  });
});
