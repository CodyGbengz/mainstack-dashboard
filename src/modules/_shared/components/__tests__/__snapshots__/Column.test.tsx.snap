// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`COMPONENT TEST: Column > Should match snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-bdfCkl hPfXtK"
    />
  </div>
</body>
`;
